### PR TITLE
Addresses #205: Changes array for EditionInfo to JSX.Element

### LINKED ISSUE #205: Update EditionCard to take JSX.Element > array of strings
### Component in question:
- EditionCard

### Description of issue:
EditionCard currently takes an array of strings to generate the middle items of the card. Taking a JSX element would allow for this card to be more versatile.

### PR DESCRIPTION
**Pull request by helenvholmes**
## **This PR does the following:**
- Changes EditionInfo inside EditionCard to accept a JSX.Element instead of an array

### Front End Review:
- [ ] View the [EditionCard stories](http://localhost:9001/?path=/story/edition-card--edition-card-with-full-data)

### CODE REVIEW

**Comment by hokei** [comment]
A note:  This is another breaking change that would require a version bump.  Right now this would be merged around the same time as `Link` so I can make all of those changes in DRB at once, but we should be careful going forward on how to do breaking changes thoughtfully.

File: src/react-components/src/components/EditionCard/EditionCard.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export interface EditionCardProps {
     coverUrl: string;
 
     editionHeadingElement: JSX.Element;
```
Comment (hokei) [inline src/react-components/src/components/EditionCard/EditionCard.tsx]: For this to help Kristo's work, `editionHeadingElement` has to be made optional

**Comment by hokei** [comment]
I'd put a story up for a header-less edition card and have it also be an example of passing in something else (maybe an icon? may a link?) into the body of the card.  

edit: actually, we probably shouldn't because it's not explicitly ok by the design system.

File: src/react-components/src/__tests__/components/EditionCard-test.tsx [hunk 2]
```diff
@@ -70,11 +75,13 @@ describe("EditionCard", () => {
...
             coverUrl="https://placeimg.com/300/400/arch"
             editionHeadingElement={<a href="edition-link">2004 Edition</a>}
-            editionInfo={[
-                <>Published in New York by Random House</>,
-                <>Written in English</>,
-                <>Under Creative Commons License</>,
-            ]}
+            editionInfo={
+                <>
+                    <p>Published in New York by Random House</p>,
```
Comment (hokei) [inline src/react-components/src/__tests__/components/EditionCard-test.tsx:80]: is `<p>` the right tag to use here?  Maybe they should just be spans.

**Review by hokei — CHANGES_REQUESTED** [review]
see comments inline

edit:  Just the changelog and optional editionHeadingElement thing

**Comment by helenvholmes** [comment]
@hokei I know when it was just us there was a desire to get through as many breaking changes as soon as possible after our conversation with Kristo—is that no longer the case?

**Comment by hokei** [comment]
@scooooooooby  There was no changelog update here so I just wanted to be on the same page re: yes, this is a breaking change, and yes, we are making this breaking change intentionally and have thought about the work this generates for SFR.  

(if you update the changelog as a part of PR creation, I think that would make it clear and I can stop nagging about it every time haha)

**Comment by helenvholmes** [comment]
@hokei yes! And sorry that you have been having to do that, it has been in the backlog to update Travis to say something when I inevitably forget over and over 😥

**Comment by hokei** [comment]
haha no worries, Imma just nag over and over.  (This can be released as 0.4.0 since there's not going to be a release between the link stuff and when this gets in - my hope is that 0.4.0 can be the no-Lerna release but that depends on how SFR deploy goes )

**Comment by helenvholmes** [comment]
@hokei, Kristo left a comment in #206 I've also been considering, but would be a more involved update than what I've done here. Do you think I should address #206 with this work now or bundle this breaking change with the Link breaking change with a potential future breaking change?

**Comment by hokei** [comment]
I think the right way to go is to create a `Card` component - which, sadly, makes this work moot.  This wouldn't represent a breaking change for `EditionCard` but when we remove `EditionCard` that would get bundled with other breaking changes

**Comment by helenvholmes** [comment]
@hokei I think you're right. I'm going to close this out and update #206 to capture that work.

**Pull request closed without merging**